core/trino-parser/src/main/java/io/trino/sql/tree/Insert.java

```diff
@@ -29,12 +29,16 @@ public final class Insert
     private final Query query;
     private final Optional<List<Identifier>> columns;
 
+    @Deprecated
     public Insert(Table table, Optional<List<Identifier>> columns, Query query)
     {
-        this(Optional.empty(), table, columns, query);
+        super(Optional.empty());
+        this.table = requireNonNull(table, "target is null");
```

Comment:
The following constructor also needs fix.
```suggestion
        this.table = requireNonNull(table, "table is null");
```